### PR TITLE
replace data-only containers with named volumes

### PR DESCRIPTION
**Pull request by sameersbn**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,22 @@
 version: '2'
 
+volumes:
```
Comment (prydonius) [inline docker-compose.yml]: should this be below services?

Comment (sameersbn) [inline docker-compose.yml]: The location does not matter. It just made sense to me to declare the volume before it is used in a service.

Comment (sameersbn) [inline docker-compose.yml]: should I move it toward the end?

Comment (prydonius) [inline docker-compose.yml]: I think it's fine :)

**Comment by sameersbn** [comment]
change has already been made in the master...

**Pull request closed without merging**